adapters/rubicon/rubicon_test.go

```diff
@@ -957,18 +865,16 @@ func CreatePrebidRequest(server *httptest.Server, t *testing.T) (an *RubiconAdap
 		Max:     2000,
 	}, cacheClient, &hcc)
 	pbReq.IsDebug = true
-	if err != nil {
-		t.Fatalf("ParsePBSRequest failed: %v", err)
-	}
-	if len(pbReq.Bidders) != 1 {
-		t.Fatalf("ParsePBSRequest returned %d bidders instead of 1", len(pbReq.Bidders))
-	}
-	if pbReq.Bidders[0].BidderCode != "rubicon" {
-		t.Fatalf("ParsePBSRequest returned invalid bidder")
-	}
 
-	ctx = context.TODO()
+	assert.Nil(t, err, "ParsePBSRequest failed: %v", err)
+
+	assert.Equal(t, 1, len(pbReq.Bidders),
+		"ParsePBSRequest returned %d bidders instead of 1", len(pbReq.Bidders))
 
+	assert.Equal(t, "rubicon", pbReq.Bidders[0].BidderCode,
+		"ParsePBSRequest returned invalid bidder")
+
+	ctx = context.TODO()
```

Comment:
I know this is not a new change as part of this PR but still, what's the reason behind setting the `ctx` at the end of the function?

Reply:
`ctx` here is returned value.